recipes-support/lvm2/lvm2_%.bbappend

```diff
@@ -6,6 +6,10 @@ SRC_URI += " \
     file://volatiles.99_cryptsetup \
 "
 
+# OpenXT does not use LVM thin-provisioning and dropping this dependency
+# accelerates the build by avoiding a lengthy compilation of boost.
+PACKAGECONFIG_remove = "thin-provisioning-tools"
```

Comment:
This commit seems to be independent of the PR?

Reply:
Yes I think it is. Am running a build now with it removed and will drop it from the PR if it looks ok

Reply:
Builds fine without it, will drop the commit from this PR.

Reply:
It's a perfectly fine commit to go in independently.